lib/cdo/rack/request.rb

```diff
@@ -107,3 +103,5 @@ def user_id_from_session_cookie
     end
   end
 end
+Rack::Request.prepend Cdo::RequestExtension
+ActionDispatch::Request.prepend(Cdo::RequestExtension) if defined?(ActionDispatch::Request)
```

Comment:
Rails 5 uses `ActionDispatch::Request` as its request object which no longer inherits from `Rack::Request`, so we need to include our request-class extensions in both places (`Rack::Request` will continue to be used by Pegasus/Sinatra).